server/src/main/scala/services/graphstages/VoyageManifestsGraphStage.scala

```diff
@@ -113,7 +118,7 @@ class VoyageManifestsGraphStage(bucketName: String, portCode: String, initialLas
         lastSeenFileName = latestFileName
         ManifestsFeedSuccess(DqManifests(latestFileName, maybeManifests.flatten), SDate.now())
       case Failure(t) =>
-        log.warn(s"Failed to fetch new manifests: $t")
+        log.warn(s"Failed to fetch new manifests: ${t.getStackTrace}")
```

Comment:
log.warn("Failed to fetch new manifests", t)

Reply:
good catch. i've left that in there from debugging